docs/static/monitoring/pipeline-viewer.asciidoc

```diff
@@ -2,141 +2,72 @@
+[role="screenshot"]
+image::static/monitoring/images/pipeline-tree.png[Pipeline Viewer]
 
 [float]
-==== View the pipeline diagram
+==== Prerequisites
 
-To view the pipeline diagram:
+Before using the pipeline viewer:
 
-. In Logstash, start the Logstash pipeline that you want to monitor.
-+
-Assuming that you've set up Logstash monitoring, Logstash will begin shipping
-metrics to the monitoring cluster.
+* <<monitoring-logstash,Configure Logstash monitoring>>.
+* Start the Logstash pipeline that you want to monitor.
 
-. Navigate to the Monitoring tab in Kibana.
-+
-You should see a Logstash section.
-+
-[role="screenshot"]
-image::static/monitoring/images/monitoring-ui.png[Monitoring UI]
+Logstash begins shipping metrics to the monitoring cluster.
+
+[float]
+==== View the pipeline
+
+To view the pipeline:
+
+* Kibana -> Monitoring -> Logstash -> Pipelines
 
-. Click the *Pipelines* link under Logstash to see all the pipelines that are
-being monitored.
-+
 Each pipeline is identified by a pipeline ID (`main` by default). For each
-pipeline, you'll see charts showing the pipeline's throughput and the number
+pipeline, you see the pipeline's throughput and the number
 of nodes on which the pipeline is running during the selected time range.
-+
-[role="screenshot"]
-image::static/monitoring/images/pipeline-viewer-overview.png[Pipeline Overview]
-+
-// To update the screenshot above, see pipelines/tweets_about_rain.conf
-+
-
-. Click a pipeline in the list to drill down and explore the pipeline
-diagram. The diagram shows the latest version of the pipeline. To view an 
-older version of the pipeline, select a version from the drop-down list next
-to the pipeline ID at the top of the page.
-+
-NOTE: Each time you modify a pipeline, Logstash generates a new version. Viewing
-different versions of the pipeline stats allows you see how changes to the pipeline
-over time affect throughput and other metrics. Note that Logstash stores multiple
-versions of the pipeline stats; it does not store multiple versions of the pipeline
-configurations themselves.
-+
-The diagram shows all the stages feeding data through the pipeline. It also shows
-conditional logic.
-+
-[role="screenshot"]
-image::static/monitoring/images/pipeline-diagram.png[Pipeline Diagram]
-+
-// To update the screenshot above, see pipelines/tweets_about_rain.conf
-+
-The information displayed on each vertex varies depending on the plugin type.
-+
-Here's an example of an *input* vertex:
-+
-[role="screenshot"]
-image::static/monitoring/images/pipeline-input-detail.png[Input vertex]
-+
-The *I* badge indicates that this is an input stage. The vertex shows:
-+
---
-* input type - *stdin*
-* user-supplied ID - *logfileRead*
-* throughput expressed in events per second - *0.7 e/s*
 
-Here's an example of a *filter* vertex.
+Many elements in the tree are clickable. 
+For example, you can click the plugin name to expand the detail view.
```

Comment:
not sure if it matters but we refer to the component as the `Detail Drawer`, but I don't know if that's public-facing.